training-react/tic-tac-toe/src/app/screens/Login/styles.scss

```diff
@@ -0,0 +1,60 @@
+@import 'variables/colors';
+@import 'reusable/flexbox';
+
+@import url(https://fonts.googleapis.com/css?family=Roboto:300);
+
+.login-page {
+  composes: d-flex j-center a-center;
+  width: 100%;
+  height: 100vh;
+}
+
+.form {
+  position: relative;
+  z-index: 1;
+  background: #FFFFFF;
+  max-width: 450px;
+  padding: 45px;
+  text-align: center;
+  box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.2), 0 5px 5px 0 rgba(0, 0, 0, 0.24);
+
+  .img-login {
+    width: 80%;
+    margin-bottom: 16px;
+  }
+
+  button {
```

Comment:
Isn't a good practice to apply styles to general HTML elements. You could use classes